dipy/core/gradients.py

```diff
@@ -255,14 +255,11 @@ def tau(self):
 
     @auto_attr
     def qvals(self):
-        tau = self.big_delta - self.small_delta / 3.0
-        return np.sqrt(self.bvals / tau) / (2 * np.pi)
+        return np.sqrt(self.bvals / self.tau) / (2 * np.pi)
```

Comment:
I don't think that `self.tau` is defined anywhere

Reply:
self.tau is defined here: https://github.com/dipy/dipy/blob/85f62188ea53b3d1e4da31773b0202116d9470ae/dipy/core/gradients.py#L252-L254

Reply:
Yep - you are right! Sorry for missing that.